_docs/style/text-conventions/general-guidelines.md

```diff
@@ -0,0 +1,194 @@
+---
+title: General writing guidelines
+layout: docs
+parent: Text Conventions
+category: Style
+usage: |
+  These general writing guidelines includes rules for how to write instructions and descriptions that appear in a Control Panel. Consult these guidelines to write text that is clear, helpful, and grammatically correct.
+resource: false
+status: complete
+last-modified: 2018-06-01
+---
+
+{% include toc.html %}
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+## Write with active voice
+
+Active voice makes the performer of the action (usually the user) the subject of the sentence. Passive voice makes the recipient of the action (not the performer) the subject of the sentence.
+
+Active-voice sentences are more engaging, less complicated, less wordy, and easier to understand than passive-voice sentences. With active voice, the actions and responses of the user are easier to distinguish from the actions and responses of the technology.
+
+{% figure [caption:"Active voice examples"] [class:"image bg-light border"] %}
+| Use (active) | Avoid (passive) |
+| --- | --- |
+| After you install the software, start the computer. | After the software has been installed, the computer can be started. |
+| Click **OK** to save the configuration. | The configuration is saved when the **OK** button is clicked. |
+| Create a server. | A server is created by you. |
+| Rackspace products and services solve your business problems. | Your business problems are solved by Rackspace products and services. |
+{: class="hxTable"}
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section"  markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+## Write clearly and objectively
+
+Use the following guidelines to help you write clearly and objectively:
+
+- Choose language that is clear and easy to understand. When possible, use fewer words. Every word should demonstrate value to the user.
+- Help users avoid errors and roadblocks. Make sure content helps users recover quickly if those things happen.
+- Alert users if they can't undo changes. Explain consequences and alternatives.
```

Comment:
s/Alert users if they can't undo changes. /Inform users if they can't undo changes before they make the change.